test/main.jl

```diff
@@ -107,15 +108,39 @@ end
     CondaPkg.resolve(force=true)
     CondaPkg.rm("libstdcxx-ng", resolve=false)
     CondaPkg.resolve(force=true)
+    @test true
 end
 
-@testitem "gc()" begin
+@testitem "external conda env" begin
     include("setup.jl")
-    if testgc
-        CondaPkg.gc()
+    dn = string(tempname(), backend, Sys.KERNEL, VERSION)
+    @show dn
+    isnull || withenv("JULIA_CONDAPKG_ENV" => dn) do
+        # create empty env
+        CondaPkg.resolve()
+        @test !occursin("ca-certificates", status())
+        # add a package to specs and install it
+        CondaPkg.add("ca-certificates"; interactive=true, force=true)  # force: spurious windows failures
```

Comment:
Using `add` followed by `resolve(; force=true)` seems to fix the `windows` sporadic failures.

Reply:
Something to investigate separately, thanks.